.gitignore

```diff
@@ -1,2 +1,5 @@
 /config.yaml
+/opa-conf.yaml
```

Comment:
Is this a default file name of some sort...?

Reply:
Heh, it's the one used by Styra DAS, which is what I used for testing.. as close to a "standard" you'll get :P